var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -604,6 +611,9 @@ def url_for_version(self, version):
     depends_on("eigen@3:", when="+machdyn")
     depends_on("py-cython", when="+mliap+python")
     depends_on("py-cython", when="+ml-iap+python")
+    depends_on("py-pip", when="+python", type="build")
+    depends_on("py-wheel", when="+python", type="build")
+    depends_on("py-build", when="+python", type="build")
     depends_on("py-numpy", when="+python")
     depends_on("py-mpi4py", when="+python+mpi")
     depends_on("py-setuptools", when="@20220217:+python", type="build")
```

Comment:
setuptools 42+ required now

Reply:
Setuptools is also required at run-time due to a pkg_resources import

Reply:
Added as run dependency and added `@42:` version specifier.